L1Trigger/L1TGlobal/src/AXOL1TLCondition.cc

```diff
@@ -90,11 +90,12 @@ const bool l1t::AXOL1TLCondition::evaluateCondition(const int bxEval) const {
   int useBx = bxEval + m_gtAXOL1TLTemplate->condRelativeBx();
 
   //HLS4ML stuff
-  std::string AXOL1TLmodelversion = m_AXOL1TLmodelversion;
+  std::string AXOL1TLmodelversion = m_AXOL1TLmodelversion;  //config loading method
+  // std::string AXOL1TLmodelversion = "L1Trigger/L1TGlobal/test/GTADModel_v3"; //for .so file in test dir loading method
   hls4mlEmulator::ModelLoader loader(AXOL1TLmodelversion);
   std::shared_ptr<hls4mlEmulator::Model> model;
   model = loader.load_model();
-  cout << "loading model... " << AXOL1TLmodelversion << std::endl;
+  // cout << "loading model... " << AXOL1TLmodelversion << std::endl;
```

Comment:
Could you remove the commented code if it doesn't really serve to demonstrate something please?